docs/moderne-cli/README.md

```diff
@@ -191,69 +275,85 @@ Now that the repository is configured, it's time to migrate it to Spring Boot 3
 
    ```diff
     diff --git a/pom.xml b/pom.xml
-    index 0b8f9c2..738ff94 100644
+    index 0b8f9c2..56693d7 100644
```

Comment:
I copied over the latest diff I got because there were some minor changes between the one that was in the doc and the latest run.